config/302-pac-configmap.yaml

```diff
@@ -44,18 +44,9 @@ data:
   # Use -1 for unlimited lines.
   error-detection-max-number-of-lines: "50"
 
-  # The label or annotations of the tasks to detect simple errors
-  # simple errors are of the makefile/grep format for example :
-  #
-  # filename:line:column: error message
-  #
-  # most linter with the "Code Quality" label in the tekton hub will output
-  # in that simple format.
-  error-detection-simple-filter-to-task-labels: "Code Quality"
```

Comment:
need to send a patch to pipeline to make it works first (it doesn't pass down the task labels in status)